evalml/tests/data_checks_tests/test_target_leakage_data_check.py

```diff
@@ -46,39 +47,38 @@ def test_target_leakage_data_check_init():
 
 
 def test_target_leakage_data_check_warnings():
-    y = pd.Series([1, 0, 1, 1])
+    y = pd.Series(range(30))
     X = pd.DataFrame()
     X["a"] = y * 3
     X["b"] = y - 1
     X["c"] = y / 10
-    X["d"] = ~y
-    X["e"] = [0, 0, 0, 0]
-    y = y.astype(bool)
+    X["d"] = y % 2
+    X["e"] = [0] * 30
+    X.ww.init(logical_types={"d": "Boolean"})
```

Comment:
This seems to be related to this bug: https://github.com/alteryx/woodwork/issues/1404

Reply:
good catch - was having a lot of trouble with this one..